### PR TITLE
Secure seed compare

### PR DESCRIPTION
**Pull request by mvondracek**
Implementation of `_secure_seed_compare`.

I've just created new issue #16 concerning use of `hmac` module from Python's standard library.

### CODE REVIEW

**Comment by mvondracek** [comment]
Hi, @lsolodkova @sobuch, can you please do the review when you are available?

**Review by lsolodkova — DISMISSED** [review]

File: PA193_mnemonic_Slytherin/mnemonic.py
Comment (sobuch) [inline PA193_mnemonic_Slytherin/mnemonic.py]: can this TODO be removed now?

File: PA193_mnemonic_Slytherin/mnemonic.py
Comment (sobuch) [inline PA193_mnemonic_Slytherin/mnemonic.py]: seems to me this should be covered by __is_valid_seed, either here or before the function is called. Or is this different?

Comment (mvondracek) [reply PA193_mnemonic_Slytherin/mnemonic.py]: Function `__is_valid_seed` currently does not check types during runtime. Type hints are for static analysis, but are not enforced during runtime. This is here because of `hmac.compare_digest`, as written in the comments.

However, there would be benefits of using entropy, mnemonic, and seed as classes, but we can get to it in next steps. :)

File: PA193_mnemonic_Slytherin/mnemonic.py
Comment (mvondracek) [reply PA193_mnemonic_Slytherin/mnemonic.py]: Sure, I'll do it. :)

File: PA193_mnemonic_Slytherin/mnemonic.py
Comment (sobuch) [reply PA193_mnemonic_Slytherin/mnemonic.py]: it does check types here https://github.com/mvondracek/PA193_mnemonic_Slytherin/pull/21

Comment (mvondracek) [reply PA193_mnemonic_Slytherin/mnemonic.py]: OK, but that's not merged yet.

Comment (sobuch) [reply PA193_mnemonic_Slytherin/mnemonic.py]: Of course, we should unify our styles before changes are merged to dev though. Checks currently differ between pull requests in this sense.

Comment (mvondracek) [reply PA193_mnemonic_Slytherin/mnemonic.py]: We can wait until #21 is merged, I will then sync changes.

**Review by sobuch — DISMISSED** [review]

**Comment by mvondracek** [comment]
Waits until #21 is merged.

**Review by sobuch — DISMISSED** [review]

**Review by lsolodkova — DISMISSED** [review]

**Review by sobuch — DISMISSED** [review]

File: PA193_mnemonic_Slytherin/mnemonic.py
Comment (mvondracek) [reply PA193_mnemonic_Slytherin/mnemonic.py]: Would be solved by #26.

**Comment by mvondracek** [comment]
Updated from dev by 92ced9c. Waiting for approval to merge into dev. // @sobuch @lsolodkova

**Review by lsolodkova — APPROVED** [review]

**Review by sobuch — APPROVED** [review]

**Pull request merged**
